.github/workflows/build-test-release.yml

```diff
@@ -458,11 +458,24 @@ jobs:
           make docker-buildx-telemetry-proxy
 
   publish-helm-chart:
+    strategy:
+      matrix:
+        operator-type: [ regular, eks-addon ]
+        include:
+          - operator-type: regular
+            ecr-registry: public.ecr.aws
+            kube-rbac-proxy-image: gcr.io/kubebuilder/kube-rbac-proxy
+            kube-rbac-proxy-tag: v0.13.0
```

Comment:
This looks very wrong

Reply:
will be fixed at - https://lumigo.atlassian.net/browse/RD-11301?atlOrigin=eyJpIjoiNmQ1NTNmZThjMzYwNGQ0NjljMjgzYTA0MGM2ZWY5MGQiLCJwIjoiaiJ9